packages/styled-base/src/index.js

```diff
@@ -142,11 +142,6 @@ let createStyled: CreateStyled = (tag: any, options?: StyledOptions) => {
               }
             } else {
               newProps.ref = ref
```

Comment:
`innerRef` is popular name in the react ecosystem and numerous libraries use it, IMHO it's undesirable to filter this out now as people might want to use it to get access to wrapped composite components